Documentation/object-store-crd.md

```diff
@@ -23,8 +23,8 @@ spec:
       size: 3
   dataPool:
     erasureCoded:
-      dataChunks: 6
-      codingChunks: 2
+      dataChunks: 2
```

Comment:
the description below on line 66 also mentions these parameters, if you could update that to match.

Reply:
Thank you. It's fixed now.